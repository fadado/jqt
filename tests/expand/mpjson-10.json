// XHTML entities
<!define E "$1": "$2">&
{
    &E('price, with discount, is',88)
}
// vim:sw=4:ts=4:ai:et:syntax=javascript
