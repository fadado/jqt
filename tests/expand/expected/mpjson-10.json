{
    "price, with discount, is": "88"
}

